config/config.exs

```diff
@@ -11,7 +11,8 @@ config :situation_room, SituationRoom.Repo,
 
 config :situation_room,
   ecto_repos: [SituationRoom.Repo],
-  env: Mix.env(),
-  port: System.get_env("PLUG_PORT")
+  env: Mix.env()
+
+config :situation_room, SituationRoom.Application, port: System.get_env("PLUG_PORT")
```

Comment:
Just call the ENV variable "PORT" as this is idiomatic and what most (nearly all) projects would use so users expect to set an ENV var called "PORT". We'd change the name only if/when our application needed configuration of multiple ports and each held a specific meaning.